tests/conftest.py

```diff
@@ -94,13 +94,25 @@ def app_id():
 
 
 @pytest.fixture
-def portal_mock(dbus_test_case, portal_name, required_templates, template_params, app_id) -> PortalMock:
+def portal_mock(
+    dbus_test_case,
+    portal_name,
+    required_templates,
+    template_params,
+    app_id,
+) -> PortalMock:
```

Comment:
on that note, might be best to add a `ruff` job via precommit.ci, that way we don't have to manually do this.

Reply:
Added ruff via pre-commit in https://github.com/flatpak/xdg-desktop-portal/pull/1486 and this PR now depends on that.